pkg/actuators/machine/instaces_test.go

```diff
@@ -327,6 +328,13 @@ func TestLaunchInstance(t *testing.T) {
 			}),
 			subnetErr: fmt.Errorf("error"),
 		},
+		{
```

Comment:
You are adding yet another case here but seems that `TestLaunchInstance` is not actually validating expectations. Can you put a follow up for this?

Reply:
https://github.com/openshift/cluster-api-provider-aws/pull/145